static/app/views/organizationIntegrations/integrationExternalUserMappings.tsx

```diff
@@ -31,7 +31,7 @@ class IntegrationExternalUserMappings extends AsyncComponent<Props, State> {
       [
         'members',
         `/organizations/${organization.slug}/members/`,
-        {query: {expand: 'externalUsers'}},
+        {query: {query: 'hasExternalTeams:true', expand: 'externalUsers'}},
```

Comment:
why not use `expand` like we did with `externalUsers`? should we be using `expand` for the organization team endpoint as well? I thought that we were were trying to use that model going forward

Reply:
@MeredithAnya yes, `expand` adds fields, but I want to filter members by whether they have externalUsers. Otherwise, you will get members with `externalUsers: []`. We use the `query` path param for filter queries in API historically